.travis.yml

```diff
@@ -4,13 +4,12 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PY=27
-    - CONDA_PY=34
     - CONDA_PY=35
```

Comment:
Probably because the dependencies haven't been rerendered and uploaded for py v3.6 yet